lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -550,7 +564,7 @@ section.instructor-dashboard-content-2 {
       padding: $baseline;
     }
 
-    .form-fields {
+    .form-field {
```

Comment:
We were trying to be a little more semantic, as this an individual field within the parent `.form-fields`.